tests/src/whisk/consul/ConsulHealthTests.scala

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
This test is supposed to test a full openwhisk deployment with services registered and so on. The other test is supposed to "unit" test just the client in conjunction with a consul-server.